api/v1alpha1/constants.go

```diff
@@ -14,9 +14,8 @@ const (
 	InstanceTypeAnnotation = "metal.ironcore.dev/instance-type"
 	// ForceUpdateAnnotation is used to indicate that the spec should be forcefully updated.
 	ForceUpdateAnnotation = "metal.ironcore.dev/force-update-resource"
-
 	// OperationAnnotationForceUpdateOrDeleteInProgress allows update/Delete of a resource even if it is in progress.
 	OperationAnnotationForceUpdateOrDeleteInProgress = "ForceUpdateOrDeleteInProgress"
-	// OperationAnnotationForceUpdateOrDeleteInProgress allows update/Delete of a resource even if it is in progress.
+	// OperationAnnotationForceUpdateInProgress allows update of a resource even if it is in progress.
 	OperationAnnotationForceUpdateInProgress = "ForceUpdateInProgress"
```

Comment:
not sure why we need both constants, as `OperationAnnotationForceUpdateOrDeleteInProgress` also does updates